prime-router/src/main/kotlin/fhirengine/azure/FHIRFunctions.kt

```diff
@@ -24,6 +24,7 @@ import gov.cdc.prime.router.fhirengine.engine.elrTranslationQueueName
 import org.apache.commons.lang3.StringUtils
 import org.apache.logging.log4j.kotlin.Logging
 
+const val sendQueueName = "send"
```

Comment:
Remove this and move it to `FHIREngine.kt` with the others. Then, import it above with the rest.